analysis/lap_difference_analyzer.py

```diff
@@ -14,6 +13,31 @@ def create_curve(dataframe):
     return curve
 
 
+def earth_distance(point1, point2):
+    """
+    Calculate the great circle distance between two points
+    on the earth (specified in decimal degrees)
+
+    All args must be of equal length.
+
+    """
+    lon1, lat1, lon2, lat2 = map(np.radians, [point1[1], point1[0], point2[1], point2[0]])
```

Comment:
according to PEP-8 max line length is 79 chars, you can set the limit in PyCharm so that you can see when your line is longer (this one is)